gradle.properties

```diff
@@ -1,2 +1,4 @@
 org.gradle.parallel=true
 org.gradle.caching=true
+#kotlin.daemon.jvmargs="-Xmx8g"
+org.gradle.jvmargs="-Xmx8g"
```

Comment:
Why do we need to add these 2 lines?

Reply:
I forgot to remove this--it might be needed a bit later.